jbpm-benchmarks/kieserver-performance-tests-jmh-manual/pom.xml

```diff
@@ -0,0 +1,180 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-benchmarks</artifactId>
+    <version>7.70.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kieserver-performance-tests-jmh-manual</artifactId>
+
+  <name>KIE Server Performance Tests JMH - Manual execution</name>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-jms-client-bom</artifactId>
+        <version>${version.org.wildfly}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.glassfish</groupId>
+            <artifactId>jakarta.json</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.jboss.spec.javax.jms</groupId>
+            <artifactId>jboss-jms-api_2.0_spec</artifactId>
+          </exclusion>
+        </exclusions>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>kieserver-performance-tests-jmh</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Please move the version element to the `dependencyManagement` section.

Reply:
moved the version as suggested